smf/sled-agent/config-rss.toml

```diff
@@ -26,61 +29,3 @@ rack_secret_threshold = 1
 # in their local network, using the current workaround methods in OPTE. See
 # how-to-run.adoc for details on how to determine the value for your network.
 mac = "00:0d:b9:54:fe:e4"
-
-[[request]]
```

Comment:
This is basically *the point* of this PR - this information can - and *should be* - dynamic. RSS will control it for now, and Nexus will take more responsibility in future PRs.